cmake/init.cmake

```diff
@@ -647,6 +647,6 @@ if(OPENDDS_XERCES3)
   if(NOT XercesC_FOUND)
     message(FATAL_ERROR "Could not find XercesC")
   endif()
-  get_filename_component(_opendds_xerces3_for_ace "${XercesC_INCLUDE_DIRS}" DIRECTORY)
+  get_filename_component(_opendds_xerces3_for_ace "${XercesC_INCLUDE_DIR}" DIRECTORY)
```

Comment:
@iguessthislldo this fixed the ACE build, but OpenDDS is still failing to find Xerces headers.